src/psyclone/dynamo0p3.py

```diff
@@ -5790,11 +5800,7 @@ def _upper_bound_fortran(self):
     def has_inc_arg(self, mapping=None):
         ''' Returns True if any of the Kernels called within this loop
         have an argument with INC access. Returns False otherwise. '''
-        if mapping is not None:
-            my_mapping = mapping
-        else:
-            my_mapping = FIELD_ACCESS_MAP
-        return Loop.has_inc_arg(self, my_mapping)
+        return Loop.has_inc_arg(self)
```

Comment:
Can we get rid of the `mapping` argument to `has_inc_arg` now?

Reply:
I removed the whole function, which is not necessary anymore since it only calls the function in the base class.